doc/documentation_data_format.md

```diff
@@ -1,75 +1,58 @@
 # Optimization problem data format specification
```

Comment:
Whatever editor you are using, it raged badly through the markdown. Please fix. PyCharm should do well.

Reply:
Huh? Wow, that looks bad!
Thanks, good to know! I'll directly trash that one (it is called Bear). Iinstalled the MD-plugin for PyCharm...